src/Repository/UserRepository.php

```diff
@@ -11,14 +11,6 @@
 use Symfony\Component\Security\Core\User\PasswordUpgraderInterface;
 
 
```

Comment:
Useful DocBlocks and other comments should not be removed (cf. https://docs.phpdoc.org/guide/guides/docblocks.html).